### ISSUE TITLE
Operational Data not shown in List View

### ISSUE THREAD

**Issue by Justify87** [alexanderzobnin/grafana-zabbix#1598]
Thanks for implementing zabbix operational data. It's missing in the list view though.

**Comment by alexanderzobnin** [comment]
I treat list view as a legacy, so I'm not going to add new features there, it takes a lot of time to support multiple views. But at the same time, operational data is a kind of important part of the information, so let's get this open and wait for feedback, maybe it makes sense to add it if many people use list view.

**Comment by ISnotes** [comment]
Good afternoon. In the "table" view, I also do not show live data. Even for newly fired triggers

**Comment by serrrios** [comment]
+1

**Comment by Justify87** [comment]
> I treat list view as a legacy, so I'm not going to add new features there

That's unfortunate. We use it because it's simply looks better when shown our wall mounted TV in our IT department

**Comment by alexanderzobnin** [comment]
@Justify87 yes, for the large displays it's much better view with increased font size. Ok, let's see, maybe if many people use this view, it makes sense to not put it into legacy.

**Comment by alexanderzobnin** [comment]
@ISnotes I just checked it out, seems it happens in History mode. Could you confirm?

**Comment by ISnotes** [comment]
> @ISnotes I just checked it out, seems it happens in History mode. Could you confirm?

not only. And Recent problems and problems too

**Comment by alexanderzobnin** [comment]
@ISnotes that's strange. What Zabbix version do you use?

**Comment by ISnotes** [comment]
> @ISnotes that's strange. What Zabbix version do you use?

6.4.0

**Comment by mrngm** [comment]
We're using list view as well due to its compactness and readability on a large screen. We're a bit behind on Zabbix, we're still running 5.4. The Grafana plugin version is more recent (currently latest): 4.3.1.

When inspecting the data from Zabbix with the Query inspector, we either receive empty strings, or (when manually setting operational data on a trigger), the exact input, e.g. `{ITEM.VALUE}` for the field `opdata`. According to [the documentation](https://www.zabbix.com/documentation/5.4/en/manual/web_interface/frontend_sections/configuration/hosts/triggers?hl=Operational%2Cdata), it should resolve dynamically, but perhaps only when queried from within Zabbix.

On the plugin side, when I switch to Table, and click on the I-button to expand one trigger, it does show the (probably latest?) value from the item of the trigger. Operational data was empty, however.

I'm currently leaning towards this being an omission in the Zabbix API, not in the plugin. If I look in Zabbix -> Problems, and activate 'Operational data', it also shows the (probably latest?) data.

**Comment by remote24** [comment]
We are also using list view on a 70" TV in the office and it would be very helpful to display Operational Data on it as well.
Using Zabbix 6.4.2, Grafana 10.2.1 and Plugin V4.4.3 and Operational Data is only visible in Table view.
